common/models/account.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function(Account) {
+  console.log('Account');
```

Comment:
Remove, this may confuse people as to why you're console logging account. Unless you explain why.